packages/mdc-segmented-button/_index.scss

```diff
@@ -0,0 +1,115 @@
+//
```

Comment:
This is the only changed document that needs reviewing. The others are addressed in PR #6069.